modules/gdscript/tests/scripts/runtime/features/lambda_bind_argument_count.out

```diff
@@ -0,0 +1,5 @@
+GDTEST_OK
+0
+0
+0
+0
```

Comment:
Shouldn't we also test some lambdas with actual arguments?

Reply:
General tests for arguments are done in an existing test, but can add some as well, should work just the same though as the base functionality is confirmed, but won't hurt to be thorough